src/crypto/index.ts

```diff
@@ -231,6 +231,14 @@ export enum CryptoEvent {
     KeyBackupStatus = "crypto.keyBackupStatus",
     KeyBackupFailed = "crypto.keyBackupFailed",
     KeyBackupSessionsRemaining = "crypto.keyBackupSessionsRemaining",
+    /**
+     * Fires when a new valid backup decryption key is in cache.
+     *
+     * The payload is the `KeyBackupInfo`.
+     *
+     * This event is only fired by the rust crypto backend.
+     */
+    KeyBackupPrivateKeyCached = "crypto.KeyBackupPrivateKeyCached",
```

Comment:
```suggestion
    KeyBackupPrivateKeyCached = "crypto.keyBackupPrivateKeyCached",
```

For consistency with the other events.

Also: don't we call this a "backup decryption key" rather than a "backup private key" elsewhere?

Reply:
good point renamed to keyBackupDecryptionKeyCached